sklearn/cross_decomposition/pls_.py

```diff
@@ -325,13 +328,13 @@ def fit(self, X, Y):
             Xk -= np.dot(x_scores, x_loadings.T)
             if self.deflation_mode == "canonical":
                 # - regress Yk's on y_score, then subtract rank-one approx.
-                y_loadings = (np.dot(Yk.T, y_scores)
-                              / np.dot(y_scores.T, y_scores))
+                y_loadings = (np.dot(Yk.T, y_scores) /
```

Comment:
Please do not change things that are not related to the PR. This adds some noise and makes it harder to review.